pages/resources/user.mdx

```diff
@@ -773,6 +773,7 @@ At least one active Gateway session is required to modify a user account's usern
 | avatar?                   | ?[image data](/reference#cdn-data) | The user's avatar; can be animated when the user has an applicable premium (Nitro) plan                                                                                   |
 | avatar_decoration_id?     | ?snowflake                         | The ID of the user's avatar decoration                                                                                                                                    |
 | avatar_decoration_sku_id? | ?snowflake                         | The SKU ID of the user's avatar decoration                                                                                                                                |
+| avatar_description?       | ?string                            | The user's avatar description usually in format "{filename}, added {date}" (max 1024 characters)                                                                          |
```

Comment:
```suggestion
| avatar_description?       | ?string                            | The description of the new user avatar, usually in the format "{filename}, added {date}" (max 1024 characters)                                                                          |
```
this is also missing the `avatar_id` field, also I would move this under `avatar?` so it's `avatar avatar_id avatar_description`